cli/src/cli-rpc-ops.c

```diff
@@ -1970,6 +1970,60 @@ gf_cli_ganesha_cbk(struct rpc_req *req, struct iovec *iov, int count,
     return ret;
 }
 
+static char *
+added_user_xlator(void *myframe)
+{
+    call_frame_t *frame = NULL;
+    cli_local_t *local = NULL;
+    char **words = NULL;
+    char *key = NULL;
+    char *value = NULL;
+    char *added_xlator = NULL;
+    int i = 0;
+
+    /*
+     * HACK:
+     *  The server xlator list is also defined here, but it should
+     *  have a common definition with glusterd. However, we are hesitant
+     *  to include the internal header of glusterd.  If want to use the
+     *  shared definition, we need to make a big modification.
```

Comment:
I was thinking it would be better to send the success/failure message as a key-value pair in the response dict from glusterd, but I realized with your current glusterd side changes, you can rely on `rsp.op_ret` value in  `gf_cli_set_volume_cbk()`. If it is -1, we know its a failure and if its 0, we can form the added_xlator name from the words list itself, without needing this hack or the strcmp.

Reply:
Thanks for the great idea.
I'll merge the current two separate is_server_debug_xlator and added_user_xlator into added_server_xlator and modify it to only display when the op_ret value is 0.